web/src/unread.js

```diff
@@ -798,6 +798,18 @@ export function stream_has_any_unmuted_mentions(stream_id) {
     return streams_with_mentions.has(stream_id);
 }
 
+export function stream_has_only_muted_mentions(stream_id) {
+    let stream_has_only_muted_mentions = true;
+    for (const topic of get_topics_with_unread_mentions(stream_id)) {
+        if (!user_topics.is_topic_muted(stream_id, topic)) {
+            stream_has_only_muted_mentions = false;
+            continue;
+        }
+    }
+
+    return get_topics_with_unread_mentions(stream_id).size !== 0 && stream_has_only_muted_mentions;
+}
```

Comment:
I don't like the idea of adding this function -- it adds a bunch of duplicate work. We should make the existing logic that computes `streams_with_mentions` and `stream_has_any_unmuted_unread_mention` compute this too in the single pass through the data that we're already doing.